init.lua

```diff
@@ -1333,20 +1338,162 @@ function torchcraft:get_layer(layer_type, also_enemy)
                                        32, 32, color)
             end
         end
+    elseif layer_type == "minerals" then
```

Comment:
Yeah, either make it green or keep it greyscale like in get_feature and hope the nets can infer the difference from the Type layer. I might write another function that takes a list of features and returns a single merged tensor.